Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2795,6 +2796,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 private:
     WebPageProxy(PageClient&, WebProcessProxy&, Ref<API::PageConfiguration>&&);
     void platformInitialize();
+    RefPtr<WebAutomationSession> activeAutomationSession() const;
```

Comment:
Please move this declaration down to other inspector/webdriver related method declarations.